group/templates/group/profile.html

```diff
@@ -1,6 +1,47 @@
 {% extends "main/layout.html" %}
 
 {% block body %}
-    <h1>You created Group. Good job !</h1>
-    {{form_data}}
+    {% if group.cover_photo %}
+        <img src="{{ group.cover_photo.url }}" alt="group photo">
+    {% endif %}
+
+    <h1>{{ group.name }}</h1>
+    <p>Details: {{ group.description }}</p>
+    <p>Location: {{ group.location }}</p>
+    <p>Visibility: {{ group.visibility }}</p>
+    <p>Join Mode: {{ group.join_mode }}</p>
+    <p>Capacity: {{ group.capacity }}</p>
+    <p>Created At: {{ group.created_at }}</p>
+    <p>Created By: {{ group.creator }}</p>
```

Comment:
We say details for description. for user to understand I am guessing.
This should apply to Visibility, Join Mode, Capacity, Create at as well.
we can discuss with this together.